src/Arriba/Arriba/Server/Authorization/IArribaIdentityNormalizer.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Security.Claims;
+
+namespace Arriba.Server.Authorization
+{
+    public interface IArribaIdentityNormalizer
+    {
+        /// <summary>
+        /// Get the IPrincipal with JWT claims and normalize its properties to the Arriba needs
+        /// </summary>
+        /// <param name="identity"></param>
+        /// <returns></returns>
+        ClaimsPrincipal NormalizeIdentity(ClaimsPrincipal identity, bool throwExceptionWhenNameNotFound);
```

Comment:
throwExceptionWhenNameNotFound seems like this is always true in code, besides tests. If that is the case the it could likely be removed.

Reply:
At the time I was thinking to give an option to not throw the exception.